src/admin/components/ResourceTabs/PolygonReviewTab/components/Polygons.tsx

```diff
@@ -0,0 +1,251 @@
+          deletePolygon(item);
+        }}
+      />
+    );
+  };
+
+  const polygonMenuItems = (item: any) => [
+    {
+      id: "1",
+      render: () => (
+        <div className="flex w-full items-center gap-2">
+          <Icon name={IconNames.POLYGON} className="h-6 w-6" />
+          <Text variant="text-12-bold">Edit Polygon</Text>
+        </div>
+      ),
+      onClick: () => {
+        setSelectedPolygon(item);
+        setPolygonFromMap({ isOpen: true, uuid: item.uuid });
+        setIsOpenPolygonDrawer(true);
+        setIsPolygonStatusOpen(false);
+      }
+    },
+    {
+      id: "2",
+      render: () => (
+        <div className="flex items-center gap-2">
+          <Icon name={IconNames.SEARCH_PA} className="h-6 w-6" />
+          <Text variant="text-12-bold">Zoom to</Text>
+        </div>
+      ),
+      onClick: () => {
+        flyToPolygonBounds(item);
+      }
+    },
+    {
+      id: "3",
+      render: () => (
+        <div className="flex items-center gap-2">
+          <Icon name={IconNames.DOWNLOAD_PA} className="h-6 w-6" />
+          <Text variant="text-12-bold">Download</Text>
+        </div>
+      ),
+      onClick: () => {
+        downloadGeoJsonPolygon(item);
+      }
+    },
+    {
+      id: "4",
+      render: () => (
+        <div className="flex items-center gap-2">
+          <Icon name={IconNames.COMMENT} className="h-6 w-6" />
+          <Text variant="text-12-bold">Comment</Text>
+        </div>
+      ),
+      onClick: () => {
+        setSelectedPolygon(item);
+        setIsOpenPolygonDrawer(true);
+        setIsPolygonStatusOpen(true);
+      }
+    },
+    {
+      id: "5",
+      render: () => <div className="h-[1px] w-full bg-grey-750" />,
+      MenuItemVariant: MENU_ITEM_VARIANT_DIVIDER
+    },
+    {
+      id: "6",
+      render: () => (
+        <div className="flex items-center gap-2">
+          <Icon name={IconNames.TRASH_PA} className="h-6 w-6" />
+          <Text variant="text-12-bold">Delete Polygon</Text>
+        </div>
+      ),
+      onClick: () => {
+        openFormModalHandlerConfirm(item);
+      }
+    }
+  ];
+
+  return (
+    <div>
+      <Drawer isOpen={isOpenPolygonDrawer} setIsOpen={setIsOpenPolygonDrawer} setPolygonFromMap={setPolygonFromMap}>
+        <PolygonDrawer
+          polygonSelected={selectedPolygon?.uuid || ""}
+          isPolygonStatusOpen={isPolygonStatusOpen}
+          refresh={props?.refresh}
+        />
+      </Drawer>
+      <div className="mb-4 flex items-center gap-1">
+        <Text variant="text-16-bold" className="pl-2 text-darkCustom">
+          Polygons
+        </Text>
+        <Button variant="text" onClick={() => toggleUserDrawing?.(true)}>
+          <Icon name={IconNames.PLUS_CIRCLE} className="h-4 w-4" />
+        </Button>
+      </div>
+      <div ref={containerRef} className="flex max-h-full max-h-screen flex-col overflow-auto">
+        {polygonMenu.map(item => {
+          return (
+            <div
```

Comment:
Skip `return (` statement for simple arrow functions

Reply:
Done.